lib/sentry_slack.py

```diff
@@ -0,0 +1,50 @@
+import json
+from pathlib import Path
+
+def all_available_versions():
+    versions = []
+    csv_files = Path('.').glob('*.csv')
+    for file in csv_files:
+        print(f"File: {file.name}, Suffix: {file.stem}")
+        version = file.name.split('sentry_issues_')[-1].split('.csv')[0]
```

Comment:
Minor issue: If this file doesn't exist this function will fail, let's just wrap it in a try/catch

```
def all_available_versions():
    versions = []
    csv_files = Path('.').glob('*.csv')
    for file in csv_files:
        try:
            version = file.name.split('sentry_issues_')[1].split('.csv')[0]
            versions.append(version)
        except IndexError:
            print(f"Skipped file: {file.name} (unexpected naming format, or file doesn't exist)")
    return versions
```